manifests/init.pp

```diff
@@ -156,7 +157,7 @@
   # If undefined and selinux is present and not disabled, use selinux.
   case $mrepo::selinux {
     undef: {
-      case $::selinux {
+      case $selinux {
```

Comment:
This looks incorrect and like it was also always broken.
Problem 1) You’ve switched from the `selinux` fact to the local variable of the same name.
Problem 2) The selinux core fact is actually a Boolean so would have never resolved to `enforcing` or `permissive`.

Reply:
Turns out I’ve noticed this before! https://github.com/voxpupuli/puppet-mrepo/pull/56#discussion_r91666404 :)

Reply:
I've opened https://github.com/voxpupuli/puppet-mrepo/pull/112 that should fix it .